pkg/lvm/lvm_util.go

```diff
@@ -375,6 +375,55 @@ func GetVolumeDevPath(vol *apis.LVMVolume) (string, error) {
 	return dev, nil
 }
 
+// Validate the size volume based on extents. If the number of extents are already
+// greater than or equal to required, then make a no-op.
+func validateLVExtentSizes(vol *apis.LVMVolume, desiredSizeBytes uint64) bool {
```

Comment:
function name is a bit confusing on what it is doing especially as it's returning true in case of errors
maybe something more like: `needsLvResize` ?